composer.json

```diff
@@ -31,9 +31,10 @@
     "prefer-stable": true,
     "minimum-stability": "dev",
     "require": {
-        "php": ">=7.0"
+        "php": ">=7.1"
     },
     "require-dev": {
+        "api-platform/core": "2.5",
```

Comment:
```suggestion
        "api-platform/core": "^2.5.1",
```

Reply:
Good catch ! I thought I added it with patch level... my bad.